components/camel-jbpm/src/main/java/org/apache/camel/component/jbpm/workitem/AbstractCamelWorkItemHandler.java

```diff
@@ -130,7 +155,16 @@ public void executeWorkItem(WorkItem workItem, final WorkItemManager manager) {
             }
             handleResponse(responseExchange, workItem, manager);
         } catch (Exception e) {
-            handleException(e);
+            /*
+             * Handle the exception if 'HandleException' is enabled (which is the default).
+             * If it's not enabled, simply throw the exception. Note that in that case, the
+             * exception needs to be a RuntimeException.
+             */
+            if (isHandleException) {
+                handleException(e);
+            } else {
+                throw (RuntimeException)e;
```

Comment:
are we sure this is always the case? meaning it will always be a RuntimeException....